<launch>
    <!-- Static Transform Publisher -->
    <node pkg="nrs_ver2" type="xacro_tf.py" name="xacro_tf" output="screen">
        <param name="yaml_file" value="$(find nrs_ver2)/config/params.yaml" />
        <param name="xacro_file" value="$(find nrs_ver2)/urdf/scan_model.xacro" />
    </node>
</launch>
